Sources/SwiftSyntax/gyb_generated/syntax_nodes/SyntaxStmtNodes.swift

```diff
@@ -2960,13 +2984,33 @@ public struct ReturnStmtSyntax: StmtSyntaxProtocol, SyntaxHashable {
       unexpectedAfterExpression?.raw,
     ]
     let data: SyntaxData = withExtendedLifetime(SyntaxArena()) { arena in
-      let raw = RawSyntax.makeLayout(kind: SyntaxKind.returnStmt,
-        from: layout, arena: arena)
+      let raw = RawSyntax.makeLayout(
+        kind: SyntaxKind.returnStmt, from: layout, arena: arena,
+        leadingTrivia: leadingTrivia, trailingTrivia: trailingTrivia)
       return SyntaxData.forRoot(raw)
     }
     self.init(data)
   }
 
+  public init(
+    leadingTrivia: Trivia? = nil,
+    _ unexpectedBeforeReturnKeyword: UnexpectedNodesSyntax? = nil,
+    returnKeyword: TokenSyntax = .returnKeyword(),
+    _ unexpectedBetweenReturnKeywordAndExpression: UnexpectedNodesSyntax? = nil,
+    _ unexpectedAfterExpression: UnexpectedNodesSyntax? = nil,
```

Comment:
Not sure how I feel about this. Should we just have the one and if so, which one? I think my preference would be to skip `unexpectedBetweenReturnKeywordAndExpression`.